configapp/configapp.py

```diff
@@ -18,6 +18,12 @@
 predict_dir = os.path.join(filedir, app.config['PREDICTION_PATH'])
 
 
+@app.context_processor
+def inject_picture_size():
```

Comment:
@Adman While we are at it, do you think it would make sense to let the frontend know some other `app.config` variables? (for instance `DEBUG` comes to mind right away).

Do you think it would make sense to push this even further, and create a JS class that would make this info available to the frontend?